.phpunit.result.cache

```diff
@@ -0,0 +1 @@
ectory":0,"josegonzalez\\Dotenv\\LoaderTest::testParseFileIsUnreadable":0,"josegonzalez\\Dotenv\\LoaderTest::testFilters":0,"josegonzalez\\Dotenv\\LoaderTest::testSetFilters":0.001,"josegonzalez\\Dotenv\\LoaderTest::testSetFilterNonexistentFilter":0,"josegonzalez\\Dotenv\\LoaderTest::testSetFilterInvalidCallable":0,"josegonzalez\\Dotenv\\LoaderTest::testFilter":0.009,"josegonzalez\\Dotenv\\LoaderTest::testFilterCallable":0.002,"josegonzalez\\Dotenv\\LoaderTest::testLowercaseKeyFilter":0.002,"josegonzalez\\Dotenv\\LoaderTest::testNullFilter":0.001,"josegonzalez\\Dotenv\\LoaderTest::testRemapKeysFilter":0.002,"josegonzalez\\Dotenv\\LoaderTest::testUppercaseFirstKeyFilter":0.002,"josegonzalez\\Dotenv\\LoaderTest::testUrlParseFilter":0.002,"josegonzalez\\Dotenv\\LoaderTest::testUnderscoreArrayFilter":0.003,"josegonzalez\\Dotenv\\LoaderTest::testMultipleFilters":0.001,"josegonzalez\\Dotenv\\LoaderTest::testExpect":0.001,"josegonzalez\\Dotenv\\LoaderTest::testExpectRequireParse":0,"josegonzalez\\Dotenv\\LoaderTest::testExpectLogicException":0.001,"josegonzalez\\Dotenv\\LoaderTest::testExpectRuntimeException":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToApacheSetenvExceptionUnavailable":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToApacheSetenv":0.028,"josegonzalez\\Dotenv\\LoaderTest::testToApacheSetenvSkip":0.003,"josegonzalez\\Dotenv\\LoaderTest::testToApacheSetenvException":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToApacheSetenvPreserveZeros":0.003,"josegonzalez\\Dotenv\\LoaderTest::testDefine":0.001,"josegonzalez\\Dotenv\\LoaderTest::testDefineSkip":0.001,"josegonzalez\\Dotenv\\LoaderTest::testDefineException":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToPutenv":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToPutenvSkip":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToPutenvException":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToPutenvPreserveZeros":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToEnv":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToEnvSkip":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToEnvException":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToEnvPreserveZeros":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToServer":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToServerSkip":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToServerException":0.001,"josegonzalez\\Dotenv\\LoaderTest::testToServerPreserveZeros":0.002,"josegonzalez\\Dotenv\\LoaderTest::testSkipExisting":0,"josegonzalez\\Dotenv\\LoaderTest::testPrefix":0,"josegonzalez\\Dotenv\\LoaderTest::testToArray":0.002,"josegonzalez\\Dotenv\\LoaderTest::testToArrayRequireParse":0,"josegonzalez\\Dotenv\\LoaderTest::testToString":0.003,"josegonzalez\\Dotenv\\LoaderTest::testRequireParse":0.002,"josegonzalez\\Dotenv\\LoaderTest::testRequireParseException":0,"josegonzalez\\Dotenv\\LoaderTest::testRequireParseNoException":0,"josegonzalez\\Dotenv\\LoaderTest::testRaise":0,"josegonzalez\\Dotenv\\LoaderTest::testRaiseNoException":0,"josegonzalez\\Dotenv\\LoaderTest::testStatic":0.006}}
```

Comment:
Do we need this cache file?

Reply:
No, it got included by accident. I'm still working on this, it should be in draft.